boards/FSM/Src/Io/Io_SoftwareWatchdog.c

```diff
@@ -13,7 +13,7 @@ void Io_HardwareWatchdog_Refresh(void)
 
 void Io_SoftwareWatchdog_TimeoutCallback(SoftwareWatchdogHandle_t watchdog)
 {
-    BREAK_IF_DEBUGGER_CONNECTED();
+    //    BREAK_IF_DEBUGGER_CONNECTED();
```

Comment:
Forgot about this line, just asked Kiron what he thinks of this change

Reply:
Removed.